flusher/flusher/restake_db.py

```diff
@@ -0,0 +1,30 @@
+import sqlalchemy as sa
+
+from .db import metadata, Column, CustomDateTime
+
+restake_vaults = sa.Table(
+    "restake_vaults",
```

Comment:
Why all restake dbs have to has restake_ in front

Reply:
Do other modules need to follow?

Reply:
it starts to collide between modules. so I added it to avoid that and make it more clear.

In my opinion, 
- if it's obvious and unique like validators table -> no need to add prefix

However, let's align so we do it the same way on all modules.